dashboard/src/main/java/org/apache/uniffle/dashboard/web/config/DashboardConf.java

```diff
@@ -30,6 +30,12 @@ public class DashboardConf extends RssBaseConf {
           .defaultValue(19988)
           .withDescription("http server http port");
 
+  public static final ConfigOption<String> COORDINATOR_JETTY_ADDRESS =
+      ConfigOptions.key("coordinator.jetty.address")
```

Comment:
Is it possible that we will not use jetty in the future？Or we don’t use java to implement coordinator? How about coordinator.web.address or coordinator.address?

Reply:
> Is it possible that we will not use jetty in the future？Or we don’t use java to implement coordinator? How about coordinator.web.address or coordinator.address?

OK.